homeassistant/components/nx584/binary_sensor.py

```diff
@@ -101,6 +101,11 @@ def is_on(self):
         # True means "faulted" or "open" or "abnormal state"
         return self._zone["state"]
 
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        return {"zone_number": self._zone["number"]}
```

Comment:
This change seem unrelated to this PR. Please make a separate PR for this since it requires tests to be added.

Reply:
You are correct.  I didn't mean to commit that change.  I will remove and add to another PR.  It is related in a sense that you can get the zone number from an entity.  My apologies.